python/ray/worker.py

```diff
@@ -2043,6 +2043,12 @@ def wait(object_ids, num_returns=1, timeout=None, worker=global_worker):
     check_connected(worker)
     with log_span("ray:wait", worker=worker):
         check_main_thread()
+
+        #When Ray is run in PYTHON_MODE, all functions are run immediately,
+        #so all objects in object_id are ready.
+        if worker.mode == PYTHON_MODE:
+            return object_ids, [] 
```

Comment:
We need to handle this slightly differently based on the `num_returns` argument because the first list should have length at most `num_returns`.

So maybe we want to do `return object_ids[:num_returns], object_ids[num_returns:]`.

Reply:
I was looking at the plasma documentation and I wasn’t sure about this. I’ll change it to be at most length num_returns.